CONTRIBUTING.md

```diff
@@ -203,3 +203,20 @@ docker volume prune
 docker image prune --all --filter until=720h
 ```
 
+## Common Problems
+
+### Testing builder with `pack build` fails
+
+Run `pack build ... -v` to produce more verbose debug output.
+
+### `bazel test` fails on macOS or Windows 
+
+By default, `bazel` builds Go binaries for the current platform.  As GCP Buildpacks
+are targeted to Linux-based container images, our `.bazelrc` configures builds for
+the Linux AMD64 platform by default.
+
+To run tests on the local platform, override the `--plaforms` as
+follows:
+```sh
+bazel test --platforms="" pkg/...
```

Comment:
I think this is pretty counterintuitive and generally not a great user experience. If I `git clone` and `bazel test ...`, I should not get any failures. Is there a way to restrict a specific target to a single platform, e.g. through a build rule attribute? We could restrict the buildpack macros to linux by default.

Reply:
I don't think it's any worse than the current situation!

It looks like we might be able to do something with Bazel's _transitions_.

Reply:
Good point, I didn't realize it's broken either way.